src/org/usfirst/frc4904/robot/autonomous/strategies/AutonGearBoilerPegVision.java

```diff
@@ -0,0 +1,41 @@
+package org.usfirst.frc4904.robot.autonomous.strategies;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.GearioOuttake;
+import org.usfirst.frc4904.sovereignty.strategies.GearAlign;
+import org.usfirst.frc4904.standard.commands.RunAllSequential;
+import org.usfirst.frc4904.standard.commands.RunFor;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.WaitCommand;
+
+public class AutonGearBoilerPegVision extends CommandGroup {
+	// These constants are based on those in AutonGearLoadPegTime.
+	public static final double TIME_INITIAL_APPROACH_1 = 1.65;
+	public static final double TIME_TURN = 1.05;
+	public static final double TIME_INITIAL_APPROACH_2 = 0.3;
+	public static final double OUTTAKE_TIME_TOTAL = 3;
+	public static final double PRE_ALIGN_DELAY = 0.5;
+	public static final double POST_ALIGN_APPROACH_SPEED = -0.3;
+	public static final double POST_ALIGN_APPROACH_TIME = 0.2;
+
+	public AutonGearBoilerPegVision(boolean isBlue) {
```

Comment:
Can we also make a constructor for the enum `edu.wpi.first.wpilibj.DriverStation.Alliance`? This might be too complex for this PR, but I thought I'd suggest it.